recipes/hadoop_hdfs_datanode.rb

```diff
@@ -70,6 +74,71 @@
   end
 end
 
+hadoop_log_dir =
```

Comment:
seems like a ``hadoop_log_dir(service)`` helper would be useful here... this section is duplicated in several recipes with slight variations based on service